<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>13</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::DontWrapRows</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="actorsLabel">
         <property name="text">
          <string>Actors</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="addPushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="removePushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QListWidget" name="actorsListWidget"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="systemActorLabel">
         <property name="text">
          <string>System actor</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="actorsComboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="titleTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionCount_steps"/>
    <addaction name="actionCount_keywords"/>
    <addaction name="actionFind_lines_not_starting_with_an_actor"/>
    <addaction name="actionCut_to_depth"/>
    <addaction name="actionGenerate_numbered_list"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_JSON2"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionHost"/>
    <addaction name="actionPort"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommands"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionCount_steps">
   <property name="text">
    <string>Count steps</string>
   </property>
  </action>
  <action name="actionCount_keywords">
   <property name="text">
    <string>Count keywords</string>
   </property>
  </action>
  <action name="actionFind_lines_not_starting_with_an_actor">
   <property name="text">
    <string>Find lines not starting with an actor</string>
   </property>
  </action>
  <action name="actionCut_to_depth">
   <property name="text">
    <string>Cut to depth</string>
   </property>
  </action>
  <action name="actionLoad_JSON">
   <property name="text">
    <string>Load JSON</string>
   </property>
  </action>
  <action name="actionGenerate_numbered_list">
   <property name="text">
    <string>Generate numbered list</string>
   </property>
  </action>
  <action name="actionLoad_JSON2">
   <property name="text">
    <string>Load JSON</string>
   </property>
  </action>
  <action name="actionHost">
   <property name="text">
    <string>Host</string>
   </property>
  </action>
  <action name="actionPort">
   <property name="text">
    <string>Port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
